src/test/java/net/sf/jabref/logic/importer/fetcher/IsbnFetcherTest.java

```diff
@@ -3,10 +3,10 @@
 import java.io.IOException;
 import java.util.Optional;
 
-import net.sf.jabref.Globals;
 import net.sf.jabref.logic.importer.FetcherException;
 import net.sf.jabref.model.entry.BibEntry;
 import net.sf.jabref.model.entry.BibLatexEntryTypes;
+import net.sf.jabref.preferences.JabRefPreferences;
```

Comment:
According to https://github.com/JabRef/jabref/wiki/High-Level-Documentation#package-structure, the logic should not reference the preferences.

At least, this PR is a huge step in the direction of enforcing the idea of https://github.com/JabRef/jabref/wiki/High-Level-Documentation#package-structure.